### DISCUSSION TITLE
SO 2.4.20 Installation Error

### DISCUSSION THREAD

**Discussion by BurntNoodleSoup** [Security-Onion-Solutions/securityonion#11620] [Unsupported Versions] [answered]
I'm not really good at networking or cybersecurity (Fairly new to it) I tried installing SO many times but I keep getting this error
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/149000115/5e9e75ee-5c33-4a20-8c14-6c730da3cc3c)
I removed my pihole dns as I thought it was disrupting it, but I still got this error 
also here are some details about my VM

VMware Workstation 17

Memory: 15.1GB (15500MB)
Processors: 3
Storage: 216.5GB
Network Adapter: NAT
Network Adapter 2: Bridged (Automatic)
(Also I verified the ISO)

**Comment by dougburks** [comment, marked as answer]
Looking at your screenshot, if it can't resolve the hostname, then that most likely means there is a problem with your network configuration. Double-check your IP address, subnet mask, gateway, and DNS configuration and then try again. If all else fails, you can try an airgap installation as that should be able to continue without Internet access.

**Reply by BurntNoodleSoup** [reply]
Sorry for responding late, I tried a network install with Rocky Linux 9 and everything is going well, I think I'll might try a airgap install right now. Thanks!
